herbert_core/utilities/files/del_memmapfile_files.m

```diff
@@ -1,7 +1,8 @@
 function del_memmapfile_files(filelist)
 % Delete files which are unlocked but were accessed using memmapfile
-% 
-% Is this Matlab bug?
+%
+% Is this Matlab bug? Yes, proven that it exist on Windows, at least with
+% Matlab 2022b
```

Comment:
I'm taking it this this piece of code isn't the matlab bug but it bypasses it. Could you say explicitly what the bug is and how it fixes it.